Added new options to control what frames to add to the stack trace (#11844)
getsentry/sentry-docs: docs/platforms/python/configuration/options.mdx

</ConfigKey>

<ConfigKey name="add-full-stack">

> 
> When capturing errors, the Sentry stack traces only contain the frames from when an error happens. If this option is enabled (set to `True`) all frames from the beginning of the execution will be added to the stack trace sent to Sentry.
> 
> Default value is `False`.

Reviewer: ```suggestion

When capturing errors, Sentry stack traces typically only include frames that start the moment an error occurs. But if the "add-full-stack" option is enabled (set to `True`), all frames from the start of execution will be included in the stack trace sent to Sentry. By default, this option is set to `False`.
```

</ConfigKey>

<ConfigKey name="max-stack-frames">

> This option limits the number of stack frames that will be captured. Limiting is only applied if `add_full_stack` is enabled.
> 
> Default value is `100`.

Reviewer: ```suggestion
This option limits the number of stack frames that will be captured when `add_full_stack` is enabled. The default value is `100`.
```
